test/listen.test.ts

```diff
@@ -74,16 +74,12 @@ describe.skipIf(typeof EdgeRuntime === 'string' || typeof document !== 'undefine
         ({request, channel}) => {
           expect(request.headers.authorization, 'should send token').toBe('Bearer foobar')
 
-          channel!.send({event: 'disconnect'})
-          process.nextTick(() => {
-            channel!.close()
```

Comment:
closing here will trigger an error/reconnecting state, so not needed since we only take the welcome event from the observable